Target-MVD-Mathias/Manager/UserDataManager.swift

```diff
@@ -35,6 +35,10 @@ class UserDataManager: NSObject {
     return self.getUserObject() != nil
   }
   
+  class func getUserId() -> Int {
+    return self.getUserObject()?.id ?? 0
```

Comment:
This will probably result in an error from the backend when making a request with UserID = 0, if that is the behavior that you want is ok.